.github/workflows/project.yml

```diff
@@ -53,7 +53,7 @@ jobs:
   integration-cypress:
     strategy:
       matrix:
-        version: [16, latest]
```

Comment:
don't we need the "don't land in v3" label then ?

Reply:
I won't change this. It was just weird that we still had node@16, but the reason seems to be that node@18 does not work at all.